notification/app/notification/services/guardian/GuardianNotificationSender.scala

```diff
@@ -28,35 +28,46 @@ class GuardianNotificationSender(
 )(implicit ec: ExecutionContext) extends NotificationSender {
 
   val WORKER_BATCH_SIZE: Int = 10000
-  val SQS_BATCH_SIZE: Int = 200 // max 256kb per sqs call, 1 notification ~< 1kb
+  val SQS_BATCH_SIZE: Int = 10
```

Comment:
That's a big decrease. why not 100 for example?

Reply:
Because the error I get in the logs is pretty clear:
`Maximum number of entries per request are 10. You have sent 15.`

Reply:
👌